pkg/drift/assets/assets.go

```diff
@@ -257,3 +268,22 @@ func addFolderToGraph(graph map[string]*HierarchyNodeWithChildren, folder *Hiera
 
 	return nil
 }
+
+func AssetsByName(assetsByID map[string]*HierarchyNode) map[string]*HierarchyNode {
+	assetsByName := make(map[string]*HierarchyNode)
+	for _, a := range assetsByID {
+		assetsByName[a.Name] = a
+	}
+	return assetsByName
+}
+
+func Merge(assetsA, assetsB map[string]*HierarchyNode) map[string]*HierarchyNode {
```

Comment:
Maybe document that assets in B take precedence/overwrite assets in A in the final result.

Reply:
Done